accepted/color-4-new-spaces.md

```diff
@@ -1016,28 +1031,48 @@ The individual conversion algorithms are:
 > available 'in-gamut' color. Gamut mapping is the process of finding an
 > in-gamut color with the least objectionable change in visual appearance.
 
-Gamut mapping in Sass follows the [CSS gamut mapping algorithm][css-mapping].
-This procedure accepts a color `origin`, and a [known color space]
-`destination`. It returns the result of a [CSS gamut map][css-map] procedure,
-converted back into the original color space.
+Gamut mapping color `origin`, a [known color space] `destination`, and an
+unquoted string `method`. It returns a color in `origin`'s color space.
 
 * Let `origin-space` be `origin`'s color space.
 
 * If either `origin-space` or `destination` is not a [known color space], throw
   an error.
 
-* Let `mapped` be the result of [CSS gamut mapping][css-mapping] `origin`
-  color, with an origin color space of `origin-space`, and destination of
-  `destination`.
+* Let `color` be the result of [converting] `origin` into `destination`.
+
+* If `method` is not one of the methods defined below, throw an error.
+
+* Let `mapped` be the result of running the method defined below whose name
+  matches `method`. If no such method matches, throw an error.
 
 * Return the result of [converting] `mapped` into `origin-space`.
 
+#### `local-minde`
+
+The `local-minde` gamut mapping procedure in Sass follows the 13 February 2024
+draft of CSS Color Module Level 4. It returns the result of [CSS gamut
+mapping][css-mapping] `origin` with an origin color space of `origin-space` and
+destination of `destination`.
+
+[css-mapping]: https://www.w3.org/TR/2024/CRD-css-color-4-20240213/#css-gamut-mapping-algorithm
+
 > This algorithm implements a relative colorimetric intent, and colors inside
-> the destination gamut are unchanged. Since the process is lossy, authors
-> should be encouraged to let the browser handle gamut mapping when possible.
+> the destination gamut are unchanged.
+
+#### `clip`
```

Comment:
(non actionable)
Interesting, I thought this method would have been called `clamp`, I guess both work 😮

Reply:
"clamp" is usually used for individual colors, while "clip" is used for the strategy as a whole.

Reply:
I see, that makes sense